mergify_engine/worker.py

```diff
@@ -1005,9 +1010,11 @@ async def _stream_worker_task(self, stream_processor: StreamProcessor) -> None:
                 resource=owner_login_for_tracing,
             ) as span:
                 span.set_tag("gh_owner", owner_login_for_tracing)
-                await stream_processor.consume(
-                    bucket_org_key, owner_id, owner_login_for_tracing
-                )
+                with sentry_sdk.push_scope() as scope:
+                    scope.set_tag("gh_owner", owner_login_for_tracing)
+                    await stream_processor.consume(
```

Comment:
Does this line needs to be in the `with` block, even if it's not using scope ?

Reply:
Pretty sure it does otherwise the scope is closed and then you get back to the original information.
It's a bit like how the tracer works.